requests_gssapi/__init__.py

```diff
@@ -14,7 +14,7 @@
 """
 import logging
 
-from .gssapi_ import HTTPSPNEGOAuth, REQUIRED, OPTIONAL, DISABLED  # noqa
+from .gssapi_ import HTTPSPNEGOAuth, spnego, REQUIRED, OPTIONAL, DISABLED  # noqa
```

Comment:
Should this be upper case to match the other "exported symbols"?

Reply:
Very likely true, I am not a daily Python hacker. Shall I make it uppercase?

Reply:
I think it's probably best to make it match what's already there.

Reply:
Updated all occurences to uppercase.